omf-bifrost/overture-valhalla-writer/src/lib.rs

```diff
@@ -1,3 +1,6 @@
 pub mod writer;
-pub mod osm;
-pub mod utils;
\ No newline at end of file
+pub mod utils;
+
+pub mod ffi {
```

Comment:
for the auto-generated code, you want to disable some warnings


```suggestion
#[expect(non_camel_case_types, non_upper_case_globals)]
pub mod ffi {
```

Reply:
Applied your suggestion. Good idea, makes the compiler output a lot more readable. FYI, I was asking Claude if it was possible to disable these warnings, but it didn't come up with a solution. AI can't replace us just yet, it seems ;-). Thanks